.janitor/dspace.dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM janitortechnology/ubuntu-dev
+
+# Add the project 
+ADD ../ .
```

Comment:
Have you been able to confirm that the .dockerignore is honored in this operation?

Reply:
Not yet, will do some testing and report back